### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by mfaisalnazari**
Created Migrations and seeded the data.
Created one Service

### CODE REVIEW

File: app/services/recipes_service.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class RecipesService
```
Comment (IgorrMagalhaess) [inline app/services/recipes_service.rb:1]: I love a good abstraction!!

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
 
-  # Defines the root path route ("/")
-  # root "posts#index"
-end
+Rails.application.routes.draw do
+  namespace :api do
+    namespace :v1 do
+      post 'recipes', to: 'recipes#index'
```
Comment (IgorrMagalhaess) [inline config/routes.rb:5]: I think this is okay. How do you feel about using resources?

File: db/seeds.rb [hunk 3]
```diff
@@ -7,3 +7,18 @@
...
+Nutrient.create(name: 'Magnesium', health_benefits: 'Info: Magnesium can provide benefits to the prevention and slowing of progression of things like Alzheimer’s, stroke, diabetes, hypertension, ADHD, and central nervous system functions. It has a protective effect against things like overstimulation that leads to neuronal cell death, and its relationship  with the limbic system is thought to affect the formation and progression of depression. Magnesium intake has proven to be effective at reducing the risk of depression and has been associated with lower levels of anxiety in women. Sources: https://www.sciencedirect.com/science/article/pii/S0531556522003813')
```
Comment (IgorrMagalhaess) [inline db/seeds.rb:24]: This is awesome!

**Review by IgorrMagalhaess — APPROVED** [review]
This looks great!!!

**Review by JackCSweeney — CHANGES_REQUESTED** [review]
This looks good so far to me, but are you able to add some testing to the repo for the methods you've built out so far and the validations for the models that you have set up as well?

**Pull request closed without merging**